src/lib.rs

```diff
@@ -312,6 +286,8 @@ impl<T: Send> UncheckedUnsafeArc<T> {
     }
 }
 
+unsafe impl<T: Send> Send for UncheckedUnsafeArc<T> {}
```

Comment:
Not sure if this is right, but it seemed like the intent was for this to be what is now called `Send`